core/vtk/ttkPlanarGraphLayout/ttkMergeTreeVisualization.h

```diff
@@ -247,13 +247,13 @@ class ttkMergeTreeVisualization : public ttk::MergeTreeVisualization {
 
   // Custom array
   void addCustomArray(std::string &name, std::vector<double> &vec) {
-    customArrays.push_back(std::make_tuple(name, vec));
+    customArrays.emplace_back(std::make_tuple(name, vec));
```

Comment:
I think that with `emplace_back`, you don't need the call to `std::make_tuple`(it may create a temporary variable). At least that's what seems to be suggested in https://clang.llvm.org/extra/clang-tidy/checks/modernize-use-emplace.html. This also applies to the two next methods.

Reply:
Done in the next commit!